### DISCUSSION TITLE
Crawler Status

### DISCUSSION THREAD

**Discussion by brause** [Chia-Network/chia-blockchain#17690] [Support]
Seems like "my"(as in the) crawler sort of stopped working in december. Where is the current crawler ? Is it the go repository, I am asking because it is not so straight forward as it is declared as "WIP". 

I am also getting these when running:
```
2024-03-12T13:01:33.690 seeder full_node_server           : ERROR    Non existing function: new_unfinished_block2
2024-03-12T13:01:33.691 seeder full_node_server           : ERROR    Exception: Error code: INVALID_PROTOCOL_MESSAGE ['new_unfinished_block2'] <class 'chia.util.errors.ProtocolError'>, closing connection PeerInfo(_ip=IPv6Address('2601:1c0:6a00:12a0:8c3d:2405:4bc0:6987'), _port=8444). Traceback (most recent call last):
  File "/home/chia/chia-blockchain/chia/server/ws_connection.py", line 378, in _api_call
    raise ProtocolError(Err.INVALID_PROTOCOL_MESSAGE, [message_type])
chia.util.errors.ProtocolError: Error code: INVALID_PROTOCOL_MESSAGE ['new_unfinished_block2']
```
I checked and only the crawler is running.

Thanks for the great work!

**Comment by BrandtH22** [comment]
Hey @brause , the crawler is packaged with the chia client itself so all you should need to do is update chia to the latest version and run the crawler with `chia start crawler`. 

That specific protocol message was added in 2.2.1 to help with the handling of unfinished blocks so an update to the latest version should resolve the issues. New message:
https://github.com/Chia-Network/chia-blockchain/blob/73174465a779f00ddc1b7e78b58ce8073203fb80/chia/protocols/protocol_message_types.py#L118

Download location for new version: https://www.chia.net/downloads/

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
